### ISSUE TITLE
Switch from travis to Github Actions

### ISSUE THREAD

**Issue by matthewcarbone** [AI-multimodal/aimm-post-processing#2]
Just my personal preference. Thoughts?

**Comment by zleung9** [comment]
It seems "Github Actions" does the same thing as "Circle CI" right? ( I've been using Circle CI set up by Xiaohui.)
"Github Actions" sounds native to Git though, which I personally like. So yeah, it is OK with me : ).

**Comment by matthewcarbone** [comment]
@zhul9311 I don't think so. Github Actions is native to GitHub as you say. Circle CI is a different service. I like GitHub Actions because it's one less third-party thing we need in the software.
